samples/test/v3beta1/translate_batch_translate_document.test.js

```diff
@@ -0,0 +1,80 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+const {assert} = require('chai');
+const {describe, it, before, after} = require('mocha');
+const {TranslationServiceClient} = require('@google-cloud/translate').v3beta1;
+const {Storage} = require('@google-cloud/storage');
+const cp = require('child_process');
+const uuid = require('uuid');
+
+const execSync = cmd => cp.execSync(cmd, {encoding: 'utf-8'});
+
+const REGION_TAG = 'translate_batch_translate_document';
+
+describe(REGION_TAG, () => {
+  const translationClient = new TranslationServiceClient();
+  const location = 'us-central1';
+  const bucketUuid = uuid.v4();
+  const bucketName = `translation-${bucketUuid}/BATCH_OUTPUT/`;
+  const storage = new Storage();
+
+  before(async () => {
+    const projectId = await translationClient.getProjectId();
+
+    //Create bucket if needed
+    await storage
```

Comment:
I think this will work as-is, but unless I'm misunderstanding, you'd typically just:

```
try {
  await storage.createBucket();
} catch (error) {
  if (error.code ...etc..)
}
```